packages/card/index.js

```diff
@@ -40,21 +40,18 @@ class WarpCard extends WarpElement {
     this.clickable = false;
   }
 
-  get _outerClasses() {
+  get _containerClasses() {
     return fclasses({
       [ccCard.card]: true,
       [ccCard.cardShadow]: !this.flat,
-      [this.selected ? ccCard.cardSelected : ccCard.cardUnselected]: !this.flat,
+      [ccCard.cardSelected]: !this.flat && this.selected,
       [ccCard.cardFlat]: this.flat,
       [this.selected ? ccCard.cardFlatSelected : ccCard.cardFlatUnselected]: this.flat,
     });
   }
 
-  get _innerClasses() {
-    return fclasses({
-      [ccCard.cardOutline]: true,
-      [this.selected ? ccCard.cardOutlineSelected : ccCard.cardOutlineUnselected]: true,
-    });
+  get _outlineClasses() {
+    return [ccCard.cardOutline, this.selected ? ccCard.cardOutlineSelected : ccCard.cardOutlineUnselected].join(' ');
```

Comment:
Just curious, why change the way you collect these classes? I thought it was to get rid of `fclasses` call, but we still use it in line 44.

Reply:
Basically because it feels like crossing the river for water, when all conditions are `true`. 🤷 Also I don't understand the name. So these combined makes it hard to understand the code. But I reverted this change... and the bigger picture of this can be handled later.